contracts/v2/paranets/ParanetNeuroIncentivesPool.sol

```diff
@@ -286,7 +327,13 @@ contract ParanetIncentivesPool {
     }
 
     function _checkParanetOperator() internal view virtual {
-        require(paranetsRegistry.getOperatorAddress(parentParanetId) == msg.sender, "Fn can only be used by operator");
+        (address paranetKAStorageContract, uint256 paranetKATokenId) = paranetsRegistry.getParanetKnowledgeAssetLocator(
+            parentParanetId
+        );
+        require(
+            IERC721(paranetKAStorageContract).ownerOf(paranetKATokenId) == msg.sender,
+            "Caller isn't the owner of the KA"
```

Comment:
small descriptiveness update: "Caller isn't the owner of the Paranet Knowledge Asset" @u-hubar

Reply:
Was just trying to keep messages <= 32 characters as it consumes less gas then, but we can make it more descriptive